.env

```diff
@@ -0,0 +1 @@
+API_KEY="711623ab-a860-4d3a-84d1-bfa891ff5058"
```

Comment:
You should set gitignore to the env file

Reply:
My bad! Didn't run that:
_echo '.env*' >> .gitignore_